.github/workflows/python-checks.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
```

Comment:
Removed this rule, so it allows to run this CI workflow on PRs that don't target `master`.

Reply:
Fair call